jenkins/clouds/azure/daily/policies/Jenkinsfile

```diff
@@ -0,0 +1,71 @@
+def accounts_list = ['perfscale']
+pipeline {
```

Comment:
Did you run it and verify it works ?

Reply:
yes, [Jenkins result](https://jenkins.perf.lab.eng.bos.redhat.com/view/RunTestJobs/job/cloud-governance-test-policy/15/console)